### DISCUSSION TITLE
Translations Announcements

### DISCUSSION THREAD

**Discussion by Jeffser** [Jeffser/Alpaca#132] [General]
@loulou64490 
@not-a-dev-stein 
@alexkdeveloper 
@CounterFlow64 

Hi translation team, sorry for bothering you all, if you have time please update your translations, I added descriptions for every model, now, I don't expect you all to manually translate all of that, so feel free to use Google Translate or an AI for those, just make sure that they are well written.

There's also some new labels so please fill everything so that your language is completed!

Take as long as you want to, thanks for your help

**Comment by Jeffser** [comment]
Also please be aware that the app no longer takes into account any translated line that has the tag FUZZY, please remove every fuzzy line, they look like this

```
#, fuzzy
``` 

The translation updater no longer uses fuzzy, it was messing up some translations, thanks for your help

**Comment by loulou64490** [comment]
Hey, I translated the model description with deepl, that's bad but there is too much to do. Maybe one day I will do it myself 
https://github.com/Jeffser/Alpaca/pull/134

**Comment by Jeffser** [comment]
@alexkdeveloper 
@loulou64490 
@not-a-dev-stein 
@CounterFlow64 
@olumolu 
@8ar10der 

Hi everyone sorry to bother you, I wanted to ask if you guys would like me to remove the xml file from translations

This is the release notes / flathub store front metadata file

**Reply by loulou64490** [reply]
Why ? I don't understand the concern

**Reply by olumolu** [reply]
Sure if that is needed do it.
@Jeffser 
I saw for 2 languages you have mentioned my name for 2 languages you can mention yours for any one this language.
I will keep updating.

**Reply by Jeffser** [reply]
I mean I think it is unnecessary, half of the translation files is just release notes and that might be annoying

**Reply by 8ar10der** [reply]
I agree with you to remove the release change log one. Considering it will be more and more in the future.

About the flatpak front metadata. I'm not sure what it is. If it is related to the software information, such as the description, which will be shown on the flathub website or gnome software. Then my answer is no.

**Reply by loulou64490** [reply]
ho okay I understand what we are speaking about 😂
but I have no opinion lol
It's true that it takes a long time to translate everything

**Reply by 8ar10der** [reply]
> About the flatpak front metadata. I'm not sure what it is. If it is related to the software information, such as the description, which will be shown on the flathub website or gnome software. Then my answer is no.

Okay, now I back to my laptop and check the file you mentioned (data/com.jeffser.Alpaca.metainfo.xml.in). I think yes, the release part in the xml file is not necessary to translate. But the other elements I think need to be, such as **Features**/**Disclaimer**, etc

**Comment by Jeffser** [comment]
@alexkdeveloper 
@loulou64490 
@not-a-dev-stein 
@CounterFlow64 
@olumolu 
@8ar10der 
@YusaBecerikli 
@OriginalSimon 
@MehrzweckMandala 

Hi guys, sorry to bother you, I just wanted to let you all know 2.0.0 is ready and I was hopping you guys could update your translations if you have the time of course.

No pressure, I'm going to test the app for some time before updating it on Flathub, since it is the next big update I want to make sure everything works.

Thanks for your help as always!

**Reply by MehrzweckMandala** [reply]
Hey, no worries at all - I'm happy to contribute to the project, even if it's just the German translation. This is my first time contributing to an open-source project on this scale, and it feels great to be a part of it.

Edit: The new full German translation has been completed.

**Reply by Jeffser** [reply]
Hi I added a couple of new strings, please translate them if you have the time <3

**Reply by olumolu** [reply]
> Hi I added a couple of new strings, please translate them if you have the time <3

I can't find any line extra

**Reply by Jeffser** [reply]
1869 and 2041 @olumolu

**Reply by olumolu** [reply]
I can't find 2041

**Reply by olumolu** [reply]
Pulled.

**Comment by loulou64490** [comment]
Please add the crowdin integration with github actions [https://github.com/marketplace/actions/crowdin-action](https://github.com/marketplace/actions/crowdin-action), this would simplify all, and for newcomers

**Reply by loulou64490** [reply]
I think I ever asked for this but this would be awesome

**Reply by Jeffser** [reply]
Yeah but I would need to set up a crowdin instance and I'm way too lazy for that, maybe later

**Reply by olumolu** [reply]
I don't think any new such language is left which us major maybe urdu tamil

**Reply by Jeffser** [reply]
Japanese would be cool

**Comment by Jeffser** [comment]
@alexkdeveloper 
@loulou64490 
@not-a-dev-stein 
@CounterFlow64 
@olumolu 
@8ar10der 
@YusaBecerikli 
Simon (can't @ them)
@MehrzweckMandala 
@yoseforb 
@SpaciousCoder78 

Hi guys sorry to bother you all, just wanted to let you all know I've been releasing new versions, I wanted to give you some time to update the translations before the new `2.7.0` update I plan to release this weekend.

Thanks as always for the help in making Alpaca a better app <3

**Comment by Jeffser** [comment]
@alexkdeveloper 
@Jeffser 
@loulou64490 
@not-a-dev-stein 
@CounterFlow64 
@olumolu 
@8ar10der 
@YusaBecerikli 
@OriginalSimon 
@MehrzweckMandala 
@yoseforb 
@SpaciousCoder78 
@edo0 

Hi everyone, first of all, happy holidays!

Sorry for bothering you all, I just updated all the translation files, I wanted to have every translation completed before I release 3.0.0, there's a bunch of new features and fixes and I want everyone to have a nice experience, that includes translations, I do want to release it before the 24th as a gift for the holidays though.

As always thanks for making Alpaca a better app :D

**Reply by Jeffser** [reply]
also in case you haven't seen it, I posted this form to get some usage information, if you are interested on giving feedback on Alpaca's development please fill it, thanks!

https://github.com/Jeffser/Alpaca/issues/399

**Reply by yarons** [reply]
Hey, @yoseforb asked me to assist.
Is there a localization platform?
Would you consider one?
Thanks :)

**Reply by Jeffser** [reply]
> Hey, @/yoseforb asked me to assist. Is there a localization platform? Would you consider one? Thanks :)

Hi, thanks for your help, most people use [GTranslator](https://flathub.org/apps/org.gnome.Gtranslator) to help them with the manipulation of the file.

I'm aware there are better online platforms for this but I haven't been able to set them up.

**Reply by yarons** [reply]
@Jeffser This project is probably eligible for the libre hosting plan in Weblate, I'd love to help you set it up.

**Reply by Jeffser** [reply]
I will look into that after the holidays, thanks for the suggestion

**Comment by Jeffser** [comment]
Hi everyone!

Russian @alexkdeveloper @col83 
Spanish @Jeffser 
French @loulou64490 @topiga 
Brazilian Portuguese @not-a-dev-stein @antun3s 
Norwegian @CounterFlow64 
Bengali @olumolu 
Simplified Chinese @8ar10der @Aleksanaa 
Hindi @olumolu 
Turkish @YusaBecerikli 
Ukrainian @OriginalSimon 
German @MehrzweckMandala 
Hebrew @yoseforb 
Telugu @SpaciousCoder78 
Italian @edo0 
Japanese @sh1d0re 
Dutch @Henkster72 
Indonesian @nofalbriansah 
Tamil @harimanish 
Georgian @EkaterinePapava 
Kannada @roche-jeethan 
Arabic @x9a 
Belarusian @PlagaMedicum 
Kabyle @BoFFire @ZiriSut

First of all thanks to everyone for their help through the months! Alpaca has really grown and I'm happy that people from all over the world can enjoy it!

---

Now since I haven't done any updates in translations here for a while there's a couple of things I wanted to touch on.

## Inactive Translations

Of course I'm not asking for all translations to be up to date all the time, even I sometimes forget about Spanish but there are some translations that haven't been touched in a bit, if you are unable to keep translating to your language please tell me so I can mark it as inactive and disable it.

Either way I don't mind it too much so don't worry about it if you can't work on it right now, all of this is contribution work at the end of the day.

## New Translation Workflow

For a while I've been updating translations manually for every release which has resulted in _really big_ commits, to prevent this in the future I made a new tool available in the root of the repo `translation_tool.sh`.

With this tool you can select your language and have it's corresponding file updated, that way you can also update your translations at any moment without having to wait for me to update all of them or having to do it manually.

**Reply by ghost** [reply]
Hi @Jeffser Thank you !
For kabyle language for instance,, we are using Codeberg's Weblate :) (I manage the translation in private mode for the moment and it is synched with your Git repo and main branch) on :

https://translate.codeberg.org/

![IMG_20250909_091138](https://github.com/user-attachments/assets/8a364d53-265b-4954-a37d-ba4f3da79f34)

**Comment by loulou64490** [comment]
hello, 
I don't have enough time... sorry
